tests/test-sessions/test-jdbc-sessions/src/test/java/org/eclipse/jetty/server/session/SessionTableSchemaTest.java

```diff
@@ -119,8 +117,10 @@ public void testLoad()
         _da.initialize();
         _tableSchema.prepareTables();
 
+        String id = Long.toString(System.nanoTime());
```

Comment:
Just out of interest, if the tables are dropped after every test, you don't need to make the id unique. Fine either way.

Reply:
ah yes I added that before truncate table after every tests.